source/app/deploy/rest/otus-api/activity/activity-rest-service.js

```diff
@@ -82,7 +85,7 @@
       if (!_rest) {
         throw new Error('REST resource is not initialized.');
       }
```

Comment:
Não deveria estar no início, antes de _rest.listAll?

Reply:
Sim, mas existe a condição antes.